MSMAll/MSMAllPipeline.sh

```diff
@@ -112,6 +113,37 @@ opts_ShowValues
 #  Main processing of script.
 # ------------------------------------------------------------------------------
 log_Msg "Starting main functionality"
+# default file names
+if [[ "$RegConfPath" == "" ]]
+then
+	RegConfPath="MSMAllStrainFinalconf1to1_1to3"
+fi
+
+if [[ "$MyelinTarget" == "" ]]
+then
+	MyelinTarget="Q1-Q6_RelatedParcellation210.MyelinMap_BC_MSMAll_2_d41_WRN_DeDrift.32k_fs_LR.dscalar.nii"
+fi
+
+if [[ "$RSNTemplates" == "" ]]
+then
+	RSNTemplates="rfMRI_REST_Atlas_MSMAll_2_d41_WRN_DeDrift_hp2000_clean_PCA.ica_dREPLACEDIM_ROW_vn/melodic_oIC.dscalar.nii"
+fi
+
+if [[ "$RSNWeights" == "" ]]
+then
+	RSNWeights="rfMRI_REST_Atlas_MSMAll_2_d41_WRN_DeDrift_hp2000_clean_PCA.ica_dREPLACEDIM_ROW_vn/Weights.txt"
+fi
+
+if [[ "$TopographyROIs" == "" ]]
+then
+	TopographyROIs="Q1-Q6_RelatedParcellation210.atlas_Topographic_ROIs.32k_fs_LR.dscalar.nii"
+fi
+
+if [[ "$TopographyTarget" == "" ]]
+then
+	TopographyTarget="Q1-Q6_RelatedParcellation210.atlas_Topography.32k_fs_LR.dscalar.nii"
+fi
+
 # MSMAll config file
 RegConfPath="${MSMCONFIGDIR}/${RegConfPath}"
```

Comment:
Could put an `if [[ "$RegConfPath" != /* ]]` around this, to support absolute paths.